created a credits screen.

MJTheisen opened the pull request:
I created a Credits screen similar to that of the Retry screen that was already implemented. I moved the credits to the bottom left of the screen for no particular reason. If you want, we can re-center it. I also added it to trigger on the death state on the Hive Knight Boss. That being said, I do want the boss to actually have his fade-out sequence (which looks like it's already in the code) so I'm pretty sure that I need to add in some sort of time but I forget how to do it. We can go over that in our meeting if you want. Let me know if there are any problems.

MJTheisen committed:
created a credits screen.

dapinkone committed:
wireframe of Hive_Knight so it's not busted.

dapinkone commented:
Intended to trigger on Hive_Knight death, the code never got run because Hive_Knight was not complete. I've added a commit here so that the hive knight class has enough functionality not to crash, and it spawns on level 1 for development purposes.
Will review again for credits screen + trigger functionality later today/tomorrow.

MJTheisen committed:
fixed the credits screen to display on Boss kill.

MJTheisen committed:
fixed the extra block in lvl 1

MJTheisen committed:
made transition blocks taller, more time in bb

MJTheisen committed:
fixed hornet dash pixel width

MJTheisen committed:
optimized

MJTheisen commented:
Fixed problems. Sending new PR

MJTheisen commented:
Reopened correct pull request. I think. Fixed credits screen on boss kill, put boss back in his room, fixed front pixel on Hornet Dash, fixed transition box height to allow staying in the box for more ticks, deleted extra transition box.
